Revit_Core_Engine/Convert/FromRevit.cs

```diff
@@ -199,39 +200,77 @@ public static IBHoMObject FromRevit(this HostObjAttributes hostObjAttributes, Di
 
         /***************************************************/
 
+        [Description("Convert a Revit cable tray into a BHoM cable tray.")]
+        [Input("cableTray", "Revit cable tray to be converted.")]
+        [Input("discipline", "Engineering discipline based on the BHoM discipline classification.")]
+        [Input("settings", "Revit adapter settings.")]
+        [Input("refObjects", "A collection of objects processed in the current adapter action, stored to avoid processing the same object more than once.")]
+        [Output("cableTray", "Resulted list of BHoM cable tray converted from a Revit cable tray.")]
+        public static List<IBHoMObject> FromRevit(this Autodesk.Revit.DB.Electrical.CableTray cableTray, Discipline discipline, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
```

Comment:
@pawelbaran is this part of Revit Toolkit architecture for these methods to return `List<IBHoMObject>` rather than `List<CableTray>` for example?

If it isn't, can we put specific return types on this.

If it is, then ignore this! :smile: